WordPress/src/main/java/org/wordpress/android/ui/accounts/LoginActivity.java

```diff
@@ -149,15 +152,24 @@ protected void onCreate(Bundle savedInstanceState) {
                     }
                     break;
                 case SELFHOSTED_ONLY:
+                    mUnifiedLoginTracker.setSource(Source.SELF_HOSTED);
```

Comment:
What do you think about renaming it to `LOGIN_WITH_SITE`? It's very explicit but I think that's a good thing in this case.

Reply:
So, this source is mainly used when the user is already logged in and taps the "+" button on the Site List screen. I'm afraid that if we call it `LOGIN_WITH_SITE` it could be confused with when the user taps the "Enter your site address" button in the Prologue screen.

I honestly feel like I'm being too picky with this, so feel free to leave it as `SELF_HOSTED` like it was before. Besides, I just realized that the button to add a site in the app is actually called "Add a self-hosted site", so it might make sense to leave it that way, even if it's not entirely accurate.

Reply:
I've renamed it back to `SELF_HOSTED`